views/redux/store/configureStore.js

```diff
@@ -0,0 +1,17 @@
+import { createStore, applyMiddleware } from 'redux'
+import createLogger from 'redux-logger';
```

Comment:
Еще раз:
```
[redux-logger v3] BREAKING CHANGE
[redux-logger v3] Since 3.0.0 redux-logger exports by default logger with default settings.
[redux-logger v3] Change
[redux-logger v3] import createLogger from 'redux-logger'
[redux-logger v3] to
[redux-logger v3] import { createLogger } from 'redux-logger'
```